src/utils/utils.ts

```diff
@@ -264,3 +264,11 @@ export const trackVirtualPageView = (pathname = "") => {
     site_section: ADOBE_ANALYTICS_SITE_SECTION,
   });
 };
+
+export const slugToString = (slug) => {
```

Comment:
Either set a default of an empty string or type it as a string. If it's ever `undefined`, this will fail so perhaps go the default route.